### DISCUSSION TITLE
why not publish it

### DISCUSSION THREAD

**Discussion by NebulaTechs** [blueedgetechno/win11React#770] [Q&A] [answered]

**Comment by yashash-pugalia** [comment]
where do you want us to publish it.

**Comment by NebulaTechs** [comment]
github release

**Comment by NebulaTechs** [comment]
（exe version)

**Comment by NebulaTechs** [comment]
the last release in in 2022

**Comment by yashash-pugalia** [comment]
well there hasn't been any major update since a long time still @andrewstech

**Comment by luni-moon** [comment]
Please don't submit this as a bug report, please continue this in a discussion.

**Comment by andrewstech** [comment]
The compiler is currently causing issues.

**Comment by NebulaTechs** [comment]
can you publish now

**Reply by luni-moon** [reply]
Please be patient!

**Reply by andrewstech** [reply]
> can you publish now

No.

**Comment by NebulaTechs** [comment]
BTW,I show that the connection has been reset when I access

**Reply by luni-moon** [reply]
Please provide a screenshot!

**Comment by NebulaTechs** [comment, marked as answer]
can you publish it in no time？
